pitest/src/main/java/org/pitest/mutationtest/incremental/IncrementalAnalyser.java

```diff
@@ -29,85 +29,77 @@
   private final CoverageDatabase           coverage;
   private final Map<DetectionStatus, Long> preAnalysed = createStatusMap();
 
-  public IncrementalAnalyser(final CodeHistory history,
-      final CoverageDatabase coverage) {
+  public IncrementalAnalyser(CodeHistory history, CoverageDatabase coverage) {
     this.history = history;
     this.coverage = coverage;
   }
 
   private static Map<DetectionStatus, Long> createStatusMap() {
-    final HashMap<DetectionStatus, Long> map = new HashMap<DetectionStatus, Long>();
-    for (final DetectionStatus each : DetectionStatus.values()) {
-      map.put(each, 0L);
+    Map<DetectionStatus, Long> map = new HashMap<DetectionStatus, Long>();
+    for (DetectionStatus detectionStatus : DetectionStatus.values()) {
+      map.put(detectionStatus, 0L);
     }
     return map;
   }
 
   public Collection<MutationResult> analyse(
-      final Collection<MutationDetails> mutation) {
+      Collection<MutationDetails> mutation) {
 
-    final List<MutationResult> mrs = new ArrayList<MutationResult>(
-        mutation.size());
-    for (final MutationDetails each : mutation) {
-      final Option<MutationStatusTestPair> maybeResult = this.history
-          .getPreviousResult(each.getId());
+    List<MutationResult> mutationResultSet = new ArrayList<MutationResult>(mutation.size());
+    for (MutationDetails mutationDetails : mutation) {
+      Option<MutationStatusTestPair> maybeResult = history.getPreviousResult(mutationDetails.getId());
       if (maybeResult.hasNone()) {
-        mrs.add(analyseFromScratch(each));
+        mutationResultSet.add(analyseFromScratch(mutationDetails));
       } else {
-        mrs.add(analyseFromHistory(each, maybeResult.value()));
+        mutationResultSet.add(analyseFromHistory(mutationDetails, maybeResult.value()));
       }
     }
 
-    logTotals();
+    logTotals(preAnalysed);
 
-    return mrs;
+    return mutationResultSet;
 
   }
 
-  private void logTotals() {
-    for (final Entry<DetectionStatus, Long> each : this.preAnalysed.entrySet()) {
-      if (each.getValue() != 0) {
-        LOG.fine("Incremental analysis set " + each.getValue()
-            + " mutations to a status of " + each.getKey());
+  private static void logTotals(Map<DetectionStatus, Long> preAnalysed) {
```

Comment:
Whats the point here? Why do you think is a private instance method more readable than a static method with a parameter?
That is just changing object oriented style into procedural style.